.github/workflows/rust-test.yml

```diff
@@ -107,8 +111,18 @@ jobs:
           - type: miri
             toolchain: ${{ inputs.nightly_toolchain }}
       fail-fast: ${{ inputs.fail_fast == 'true' }}
+    services:
+      azurite:
+        # Hack for conditionally enabling the service:
+        # https://github.com/actions/runner/issues/822
+        image: ${{ (inputs.service_azurite) && 'mcr.microsoft.com/azure-storage/azurite' || '' }}
+        ports:
+          - 10000:10000 # BLOB
+          - 10001:10001 # Queue
+          - 10002:10002 # Table
+        options: --health-cmd "nc -z 127.0.0.1 10000"
     steps:
-      - name: Set DATABASE_URL
+      - name: Start Postgres
```

Comment:
Can you rewrite this part to just use the service like azurite?

Reply:
Not really in scope for this PR, I think it should be a separate PR.